dashboard/app/models/game.rb

```diff
@@ -168,28 +168,28 @@ def flappy?
   end
 
   def uses_droplet?
-    name == "MazeEC" || name == "ArtistEC" || name == "Applab" || name == "StudioEC" || name == "Gamelab"
+    %w(MazeEC ArtistEC Applab StudioEC Gamelab).include? name
```

Comment:
Probably? That said, these strings aren't readily available in this file as constants, so I'm going to punt for now.